ush/convert_yaml_ewok2gdas.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
@danholdaway hopes to make this redundant with JCB, I suggest this script is not even needed anyways as of now